### DISCUSSION TITLE
Can I integrate 3rd Party components like data grids in Quasar

### DISCUSSION THREAD

**Discussion by talktejas** [quasarframework/quasar#11294] [General - Components / Directives / etc]
Hi. This may be a newbie question but Can I integrate 3rd Party components like data grids in Quasar? Like I need extra functionalities like those provided by Kendo UI etc. can I use Quasar and integrate Kendo or other similar components in the same app?

**Comment by hawkeye64** [comment]
Yes. Some are easier than others. There could potentially be CSS conflicts as well.
